.github/CODEOWNERS

```diff
@@ -1,16 +1,10 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
 #
-# require both blockchain-team-codeowners and blockchain-team to review all PR's not specifically matched below
-* @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team
-
-
-# Signer code
-# require both blockchain-team-codeowners and blockchain-team-signer to review PR's for the signer folder(s)
-libsigner/**/*.rs @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team-signer
-stacks-signer/**/*.rs @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team-signer
+# require both blockchain-team-codeowners and blockchain-team-reviewers to review all PR's not specifically matched below
+* @stacks-network/blockchain-team-codeowners @stacks-network/blockchain-team-reviewers
 
 # CI workflows
-#  require both blockchain-team and blockchain-team-ci teams to review PR's modifying CI workflows
-/.github/workflows/ @stacks-network/blockchain-team @stacks-network/blockchain-team-ci
-/.github/actions/ @stacks-network/blockchain-team @stacks-network/blockchain-team-ci
+#  require both blockchain-team-ci and blockchain-team-reviewers teams to review PR's modifying CI workflows
+/.github/workflows/ @stacks-network/blockchain-team-ci @stacks-network/blockchain-team-reviewers
+/.github/actions/ @stacks-network/blockchain-team-ci @stacks-network/blockchain-team-reviewers
```

Comment:
In this case, I think it means Aaron / Brice / Jude won't count as valid approvals, which wouldn't be what we want right?

Reply:
for these specific files, i didn't think they should be required since this path isn't blockchain/code specific - it mainly  pertains to CI changes. we can definitely require their review here if you want though

Reply:
and to be verbose - their review would still `count` towards the total reviews required, but not as a required codeowner for this path.